rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtilsTest.java

```diff
@@ -795,6 +798,31 @@ protected int assertFieldsArePopulated(JSONObject field, Patient patient, int nu
             assertEquals(field.getString(VALUE), UNIQUE_ID);
             numOfPopulatedFields++;
         }
+        else if (CovidConstants.FormFields.PATIENT_INFO_UNIQUE_ID.equals(field.getString(JsonFormConstants.KEY))) {
+            JSONArray options = field.getJSONArray(JsonFormConstants.OPTIONS_FIELD_NAME);
+            if (options.length() > 0) {
+                JSONObject childObject = options.getJSONObject(0);
+                assertEquals(childObject.getString(JsonFormConstants.TEXT), "{{sample-collection-form-original.step8.patient_info_unique_id.options.unique_id.text}}unique_id");
+                numOfPopulatedFields++;
+            }
+        }
+        else if (CovidConstants.FormFields.PATIENT_INFO_NAME.equals(field.getString(JsonFormConstants.KEY))) {
+            JSONArray options = field.getJSONArray(JsonFormConstants.OPTIONS_FIELD_NAME);
+            if (options.length() > 0) {
+                JSONObject childObject = options.getJSONObject(0);
+                assertEquals(childObject.getString(JsonFormConstants.TEXT), "{{sample-collection-form-original.step8.patient_info_name.options.patient_name.text}}patient");
+                numOfPopulatedFields++;
+            }
+        }
+        else if (CovidConstants.FormFields.PATIENT_INFO_DOB.equals(field.getString(JsonFormConstants.KEY))) {
+            JSONArray options = field.getJSONArray(JsonFormConstants.OPTIONS_FIELD_NAME);
+            if (options.length() > 0) {
+                JSONObject childObject = options.getJSONObject(0);
+                assertEquals(childObject.getString(JsonFormConstants.TEXT), "{{sample-collection-form-original.step8.patient_info_dob.options.patient_dob.text}}01-09-2020");
+                numOfPopulatedFields++;
+            }
+        }
```

Comment:
The assertions should be that the Patient fields are added here i.e. `"12345", 20, "01-09-2020"`.

Reply:
this is because currently we don't have any translation right now.

Reply:
That shouldn't be a problem, the value here does not depend on translations since we inject it ourselves. Similar to https://github.com/onaio/rdt-standard/blob/873de4736d6121979335979680f19f7876560a90/rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtilsTest.java#L794.

Reply:
fixed.